spec/prompt_spec.rb

```diff
@@ -31,6 +31,15 @@
       expect(config._pry_.is_a?(Pry)).to eq true
       expect(config.object).to eq self
     end
+
+    specify "object is Hash when current binding is a Hash" do
```

Comment:
Why `specify`? The whole file uses `it`.

Reply:
because it is easier to describe the test with `specify` than `it`. the whole file also uses `it "should .."` which is old rspec style, given expect() expectations.

Reply:
it "should"-style also limits your vocabulary and is redundant to repeat all the time. 
there's good arguments against it. rubinius for example considers it bad style.